### PR TITLE
Finished NavBar for Curriculum

### PR DESCRIPTION
**Pull request by hwong0305**
- decided to change layout to use navbar in courses and lessons.

### CODE REVIEW

File: pages/signup.tsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
 import React from 'react'
 import Card from '../components/Card'
-import Layout from '../components/Layout'
+import LandingNav from '../components/LandingNav'
 
 const Signup: React.FC = () => {
   return (
-    <Layout>
+    <div>
+      <LandingNav />
```
Comment (Wolfy64) [inline pages/signup.tsx]: Why did you removed `<Layout />`
Layout should wrap all pages (except the landing page)

File: components/Layout.tsx [hunk 2]
```diff
@@ -1,16 +1,31 @@
...
   return (
     <>
-      <LandingNav />
-      <div className="container">{props.children}</div>
+      <AppNav
+        loggedIn={loggedIn}
+        username={username}
+        firstName={firstName}
+        lastName={lastName}
+      />
```
Comment (Wolfy64) [inline components/Layout.tsx]: We might `useContext()` to pass `Auth data` to the <NavBar /> but maybe it's over-engineering

File: pages/signup.tsx (hunk 1 above)
Comment (hwong0305) [reply pages/signup.tsx]: No. The Login / Signup page share the `Nav` with the landing page,. I changed `Layout` to use the Nav for courses. My implementation makes Layout uses the `secondNav` which isn't compatible with signup and login.

File: components/AppNavBar.tsx [hunk 3]
```diff
@@ -1,29 +1,100 @@
...
+  firstName,
+  lastName
+}) => {
+  const initial =
+    firstName && lastName
+      ? firstName[0].toUpperCase() + lastName[0].toUpperCase()
+      : ''
+  if (!loggedIn) {
+    return (
+      <div className="bg-white">
```
[the remark this answers is not in the document]
Comment (songz) [reply components/AppNavBar.tsx]: At PayPal we did this and the components start having all sorts of logic inside and it became difficult to read. I'd rather have:

```
if (isLoggedIn) {
  return <NavBarAth>
}
return <NavBarWithoutAuth />
```

Which is kind of what this PR is already doing.

**Comment by hwong0305** [comment]
Need to update PR to add comment for why there are duplicate navbars.

File: yarn.lock [hunk 4]
```diff
@@ -1710,28 +1710,7 @@
     react-inspector "^4.0.0"
     uuid "^3.3.2"
 
-<<<<<<< HEAD
```
Comment (songz) [inline yarn.lock]: lol.

**Pull request merged**